src/com/datacash/training/chapter14/answers.md

```diff
@@ -0,0 +1,40 @@
+1) `->`
+
+2) A functional interface contains a single abstract method.
+
+3) A lambda expression defines the implementation of the abstract method from the functional interface.
+
+4) The two types of lambda expressions are Expression Lambdas, which are a single statement, and Block Lambdas, which contain multiple statements and a return statement.
+
+5) `(n) -> ( 9 < n && n < 21 )`
+
+6)
+```java
+interface MyTest {
+	boolean testing(int n);
+}
+```
+
+7) Question7.java
+
+8) Question8.java
+
+9) Question9.java
+
+10) If a lambda expression uses a local variable it must be final or effectively final.
+
+11) True
+
+12) A method reference is a way of referring to a method without executing it.
+
+13) When evaluated, a method reference creates an instance of the *functional interface* supplied by its target context.
+
+14) `MyClass::myStaticmethod`
+
+15) `mcObj::myInstMethod`
+
+16) Question16.java
+
+17) `Class::new`, where `Class` is the class whose constructor is being referenced.
+
+18) java.utils.function
```

Comment:
util, not utils, which would get flagged up straight away anyway. Other than that everything else looks good :+1: